docsrv.go

```diff
@@ -1,25 +1,54 @@
 package main
 
 import (
+	"context"
 	"net/http"
 	"os"
+	"strconv"
 	"time"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/src-d/docsrv/docsrv"
 )
 
+const (
+	sharedFolder = "/etc/shared"
```

Comment:
Now, the shared folder is passed down to the DocSrv service using the `Options`, so in docsrv itself it's never hardcoded. It's just hardcoded in tests.